app/controllers/orgs/rosters_controller.rb

```diff
@@ -76,7 +99,7 @@ def link
       user_id = params[:user_id].to_i
       raise ActiveRecord::ActiveRecordError unless unlinked_user_ids.include?(user_id)
 
-      current_roster_entry.update_attributes!(user_id: user_id)
+      current_roster_entry.update_attributes(user_id: user_id)
```

Comment:
`update_attribute` skips validations. Use `update` instead to _not_ skip validations. 
See https://davidverhasselt.com/set-attributes-in-activerecord/ for details.
```suggestion
      current_roster_entry.update(user_id: user_id)
```